### DISCUSSION TITLE
Using GitHub as your portfolio

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#40] [General]
# Using GitHub as your portfolio

GitHub is a powerful platform for developers to showcase their work and projects. It can be an excellent tool for landing a job, as many employers use it to search for and review potential candidates. In this post, we'll discuss how to effectively use GitHub as a portfolio to impress potential employers and increase your chances of getting hired.

https://steven-giesel.com/blogPost/2516bdf5-43f3-4940-a0c0-ee83dcaa5e55

**Comment by mehdihadeli** [comment]
Hey Steven :)
Unfortunately, most of the employer don't care to look at our GitHub, most of them are very lazy. This is my experience :D

**Reply by linkdotnet** [reply]
Well yes. That happens more than often I guess. I can just talk for myself and I do look on the GitHub Profile Page if provided

**Reply by mehdihadeli** [reply]
Yes, It is an excellent place for understanding the knowledge of a person. I hope GitHub helps more for finding better jobs for developers, But I highly recommend do open-source, mostly for learning from the community and practicing new thing that we haven't a chance to do in our work

**Reply by linkdotnet** [reply]
Maybe, similar to StackOverflow, GitHub goes in that direction. It seems like a logical thing to do.
